### PR TITLE
Cockpit 204

### PR DESCRIPTION
**Pull request by martinpitt**
Rendered preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-204.html

 - [x] Add screenshots for VM disk readonly/sharing
 - [x] Add bodhi URLs

### CODE REVIEW

**Comment by skobyda** [comment]
Screenshots added:
https://github.com/cockpit-project/cockpit/pull/12314

**Comment by martinpitt** [comment]
@skobyda : Can you please proofread the Machines parts for factual correctness? Thanks!

**Comment by martinpitt** [comment]
> skobyda | pitti: Sounds good. I would just like to change "sharing writable disks will destroy" to "sharing writable disks can destroy"

Done.

> Also do you think that functionality of automatically turning disks to shareable once you add disk to another VM should be omitted?

Added.

File: _posts/2019-10-02-cockpit-204.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+summary: Cockpit with Services and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 204.
+
+### System: Highlight failed services
+
+The initial System page shows a "System Health" table entry if there are any
+failed systemd services:
```
Comment (garrett) [inline _posts/2019-10-02-cockpit-204.md]: Perhaps:

A "System Health" entry on the System overview page shows failed services, if any exist.

File: _posts/2019-10-02-cockpit-204.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+![Failed services on System page](/images/system-failed-services.png)
+
+The navigation menu will have a similar warning icon next to the "Services"
+entry in that case.
```
Comment (garrett) [inline _posts/2019-10-02-cockpit-204.md]: Failed services also cause a warning to appear in the navigation, next to the "Services" entry.

File: _posts/2019-10-02-cockpit-204.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Sharing is only supported for disks in raw format. Note that sharing writable
+disks can easily destroy data when multiple VMs write to the disk at the same
+time; so use this feature with caution.
+
+Sharing will be enabled automatically when adding a disk to more than one VM.
+
+### Playground: Add index page
+
+The various "Playground" pages are meant for developers to test internal
```
Comment (garrett) [inline _posts/2019-10-02-cockpit-204.md]:
```suggestion
Various "Playground" pages are meant for developers to test internal
```

Comment (garrett) [reply _posts/2019-10-02-cockpit-204.md]: I'm just removing a "The" here. I've been on a quest to try to make the beginning of paragraphs start with different words. (It's too easy to start every one with "The" and is a bit repetitive to read.)

File: _posts/2019-10-02-cockpit-204.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+features of Cockpit; they are shipped in the "cockpit-tests" package. All these
+pages are now subsumed in a single "Development" menu entry that shows an index
+page, instead of a lot of individual menu entries that cluttered up the menu
+too much.
+
+The index page now also shows playground elements that weren't previously
```
Comment (garrett) [inline _posts/2019-10-02-cockpit-204.md]: I need to leave a few "The"s right? :wink:

**Comment by martinpitt** [comment]
@garrett: Squashed your image optimizations and applied your suggestions, and eliminiated another "The" (→ "That"). Many thanks! PTAL?

**Pull request merged**